helm/trustagent/charts/factory/templates/_job.tpl

```diff
@@ -37,19 +37,16 @@ spec:
             - name: COMPONENT
               value: {{ include "factory.name" . }}
           securityContext:
-            runAsUser: 503
-            runAsGroup: 500
-            readOnlyRootFilesystem: true
-            allowPrivilegeEscalation: false
+            {{- toYaml .Values.securityContext.aasManager | nindent 12 }}

```

Comment:
where are we setting these securityContext now ? also the useIDs ?

Reply:
Security contexts are set in respective values.yaml files.  https://github.com/open-edge-platform/trusted-compute/blob/a84e418b519417dd33d442000c2f9f99c48eb0be/helm/trustagent/values.yaml#L103